chart/values.yaml

```diff
@@ -66,6 +66,8 @@ controller:
     port: 9443 # port to use to serve requests
   safeMode:
     environment: ""
+    permittedUserGroups:
+      - system:authenticated
```

Comment:
Useful to use in the dev chart because it seems we have this by default in our local userinfo

Reply:
Could we add a comment on this new setting to explain what it does please? Especially it's only for disruptions, and it does not impact disruption cron and rollout.

Reply:
Yep! We will probably want to implement this for the other two resources too? But they dont have webhooks yet, so that will be a followup